src/vorta/utils.py

```diff
@@ -353,7 +357,20 @@ def parse_args():
         help='Create a backup in the background using the given profile. '
         'Vorta must already be running for this to work.',
     )
-
+    # the "development" attribute will be None if the flag is not called
+    # if the flag is called without an extra argument, the "development" attribute
+    # will be set to the value of DEFAULT_DIR_FLAG.
+    # if the flag is called with an extra argument, the "development" attribute
+    # will be set to that argument
+    parser.add_argument(
+        '--development',
+        '-D',
+        nargs='?',
+        const=DEFAULT_DIR_FLAG,
+        help='Start vorta in a local development environment. '
+        'All log, config, cache, and temp files will be stored within the project tree. '
+        'You can follow this flag with an optional path and it will store the files in the provided location.',
+    )
```

Comment:
Can you specify a `metavar` so that it doesn't say 

```
usage: vorta [-h] [--version] [--daemonize] [--create PROFILE]
             [--development [DEVELOPMENT]]
```

when running `vorta --help`?